snippets/fer1_annoted.js

```diff
@@ -24,7 +24,7 @@ opts.conf = {
 opts.vis = {
   conserv: false,
   overviewbox: false,
-  seqlogo: true
+  seqlogo: false
```

Comment:
I think "seqlogo" is broken with my changes. 
We don't need it in LD though, so I'll fix this later with a follow-up.